config/alr_config.ads

```diff
@@ -0,0 +1,4 @@
+--  Configuration for alr generated by Alire
```

Comment:
Should this folder _not_ be added to git?

Reply:
Fixed by #744

Reply:
Fixed in Alire, but they still should not be checked in here.